admin_manual/installation/system_requirements.rst

```diff
@@ -14,8 +14,8 @@ For best performance, stability and functionality we have documented some recomm
 +------------------+-----------------------------------------------------------------------+
 | Platform         | Options                                                               |
 +==================+=======================================================================+
-| Operating System | - Ubuntu 24.04 LTS                                                    |
-| (64-bit)         | - **Ubuntu 22.04 LTS** (recommended)                                  |
+| Operating System | - **Ubuntu 24.04 LTS** (recommended)                                                    |
+| (64-bit)         | - Ubuntu 22.04 LTS                                  |
```

Comment:
RST needs the table to be aligned otherwise it does not compile.
you need to adjust the spaces to match with the other rows.